### ISSUE TITLE
allow ascent.execute() with no args?

### ISSUE THREAD

**Issue by cyrush** [Alpine-DAV/ascent#292]
enable use of just `ascent_action.json` or `ascent_action.yaml` without passing a conduit node.

**Comment by mclarsen** [comment]
like `execute(const Node = Node)`?

**Comment by cyrush** [comment]
yes , like execute with empty node merged with whatever is in the actions file

**Comment by cyrush** [comment]
we can do this easily for c++ and Python, but c and fortran would need new names for the exec method w/out actions.

**Comment by cyrush** [comment]
since it is easy to just pass an empty node, I think we should not add this.
